ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/services/ClaimMigrator.java

```diff
@@ -100,14 +109,26 @@ private void migrateClaims(
         AtomicInteger failedOnUpdateMigrations
     ) {
         notMigratedClaims.parallelStream().forEach(claim -> {
-            migrationHandler.migrateClaim(
-                migratedClaims,
-                failedOnCreateMigrations,
-                failedOnUpdateMigrations,
-                updatedClaims,
-                claim,
-                user
-            );
+            if (fixDataIssues && isSettledOrJudgement(claim)) {
```

Comment:
Does this mean that claimed which are "fixed" are not migrated? is this what we want? (I kind of lack context about this so it's a n00b question :D)

Reply:
We are recycling our migration script here Silvia. In theory we should never have to use it to migrate a single claim ever again. @dharmendrak shall we make this a little more robust though to only rely on a single condition `fixDataIssues` - just worried we may otherwise migrate unwanted claims...